connect_ext_ppr/service.py

```diff
@@ -209,21 +217,22 @@ def create_ppr(ppr, context, deployment, db, client, logger):
             created_by=context.user_id,
         )
         db.add(file_instance)
-        db.commit()
+        db.flush()
 
         new_ppr = PPRVersion(
             file=file_instance.id,
             deployment=deployment.id,
             version=new_version,
-            product_version=deployment.product.version,
+            product_version=product_version,
+            description=ppr.description,
             summary=summary,
             status=status,
             created_by=context.user_id,
             **config_kwargs,
         )
         db.set_verbose(new_ppr)
         db.commit()
-        return new_ppr
+        return new_ppr, file_instance, active_configuration
```

Comment:
I know that it looks bad to return 3 values in a function.
But we are going to refactor this function anyway, so let's refactor this place with a function.

Reply:
I think that this could be solved by adding `relationship` on PPRVersion to Configurations and Files, like:
```
configuration = relationship('Configuration', foreign_keys="Configuration.id")
file = relationship('File', foreign_keys="File.id")
```
So then you can access new properties like: `new_ppr.configuration` or `new_ppr.file`

Reply:
I'm not sure about the file relation here... We already have a file link here, but I think that we should store a reference to the configuration that was used to generate the PPR. I'm ok to merge this now, but @Hairash submit a separate issue and patch to add the reference to the latest configuration in case of:
1. File is generated on the new product version appears
2. File is generated when user click on the regenerate button

Reply:
Ok agree with that!